app/scripts/controllers/serviceInstance.js

```diff
@@ -185,6 +186,7 @@ angular.module('openshiftConsole')
     var serviceResolved = function(serviceInstance, action) {
       $scope.loaded = true;
       $scope.serviceInstance = serviceInstance;
+      $scope.isMobileService = _.get(serviceInstance, 'metadata.labels', {}).mobile === 'enabled';
```

Comment:
Vitali mentioned a filter for this which I will add into the base branch so this may change